applications/FluidDynamicsApplication/tests/couette_flow_test.py

```diff
@@ -51,6 +51,12 @@ def tearDown(self):
         with KratosUnittest.WorkFolderScope(self.work_folder, __file__):
             KratosUtilities.DeleteFileIfExisting('couette_flow_test.time')
 
+    def _ExecuteCouetteFlowTest(self):
+        self.setUp()
+        self._CustomizeSimulationSettings()
+        self.runTest()
+        self.tearDown()
+
```

Comment:
Hi, just curious... Is there a specific reason why you use a seperate function to call it? Wouldn't it be possible by using default cals of unit tests to call `setUp`, `tearDown` methods?

Reply:
The point is that these were not called... This is why I created the auxiliary function.

Reply:
(it is possible that I did something wrong)

Reply:
I would say that now `setUp` and `tearDown` is being called twice
IMO is worth to check the reason, this could lead to more problems than it solved

Reply:
This what I thought but seems that they're not being called. I'll go through it.